commands/init_test.go

```diff
@@ -67,6 +67,13 @@ func TestCheckVersionMatch(t *testing.T) {
 			Expected:      false,
 			ErrExpected:   true,
 		},
+		{
+			Name:          "bad version",
+			Version:       "1.2.3",
+			ServerVersion: "1.2",
```

Comment:
nit: technically 1.2 should be a valid version IMO. Not a big deal though as we probbably never return a version like this. Right?

Reply:
You are right .. but the code parses 3 parts separated by ".". I think for our purposes this is good to keep. More like a test to lock in the behavior from server.